src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped work");
```

Comment:
Similarly, the message 'Bulldozer stopped work' could be more informative by including the class name to clearly indicate that it is the Bulldozer machine that has stopped working.